pandas/core/arrays/datetimelike.py

```diff
@@ -179,7 +179,8 @@ def strftime(self, date_format):
                'March 10, 2018, 09:00:02 AM'],
               dtype='object')
         """
-        return self._format_native_types(date_format=date_format).astype(object)
+        result = self._format_native_types(date_format=date_format, na_rep=np.nan)
```

Comment:
Could argue that `np.datetime64("NaT")` is more correct for datetimes?

Reply:
no, because ``np.nan`` is our string null

Reply:
Ah right. My mistake.